static/index.html

```diff
@@ -87,6 +91,41 @@ <h1><a href="/" tabindex="0">linus.zone</a></h1>
                   return false;
               }
           }
+          async function populateContentForExistingNote() {
+              const id = idInput.value;
+              if (id) {
+                  try {
+                      const content = await fetch(`/${id}/content`).then(resp => resp.json());
+                      if (content['type'] == "none") {
+                          if (!userTypedInput) {
+                              uriInput.value = "";
+                              noteInput.value = "";
+                          }
+                          return;
+                      }
+
+                      if (userTypedInput) {
+                          let replace = confirm(`Do you want to replace what you've typed with the existing content for id ${id}?`);
```

Comment:
Super minor nit pick: I tend to prefer using `let` only for variables that are changed later. Doesn't look like that's happening here, so I'd prefer `const replace` here.